config/initializers/feature_toggles.rb

```diff
@@ -9,3 +9,7 @@
     beta_testers.include?(user.email)
   end
 end
+
+OpenFoodNetwork::FeatureToggle.enable(:unit_price) do
+  ['development', 'staging'].include?(ENV['RAILS_ENV'])
```

Comment:
I'm asking for your review @sauloperez ! Thanks. 🙏

Reply:
yes, that also feels enough at this stage to me. Btw, FYI you can also use Ruby's [word array operator](https://stackoverflow.com/a/1274703/2838228) or Rails' [StringInquierer](https://stackoverflow.com/a/2715076/2838228). Just information, I don't mean to change it.